### ISSUE TITLE
Invalid Syntax errors in Python 3.12

### ISSUE THREAD

**Issue by ajinabraham** [quark-engine/quark-engine#710]
**Describe the bug**

```
/mobsf-ss-py3.12/lib/python3.12/site-packages/quark/utils/output.py:156: SyntaxWarning: invalid escape sequence '\l'
  descriptions = "\l".join(report_dict[parent]) + "\l"
/mobsf-ss-py3.12/lib/python3.12/site-packages/quark/utils/output.py:156: SyntaxWarning: invalid escape sequence '\l'
```

**To Reproduce**
Run quark analysis in MobSF with Python3.12

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by zinwang** [comment]
Hi, thank you for reporting this issue! We have confirmed it and will look into it.

**Comment by zinwang** [comment]
@ajinabraham 
Hi, we've pushed a patch to fix the issue and released a new version, [v24.11.2](https://github.com/quark-engine/quark-engine/releases/tag/v24.11.2). Please give it a try.

I'll close this issue. If the problem persists or you have further questions, feel free to reopen this issue.

**Comment by ajinabraham** [comment]
Thanks for the quick turnaround!
